src/kontrol/__main__.py

```diff
@@ -89,12 +145,17 @@ def main() -> None:
 
     _check_k_version()
 
+    stripped_args = {
+        key: val for (key, val) in vars(args).items() if val is not None and not (isinstance(val, Iterable) and not val)
```

Comment:
What does the condition express?

Reply:
argparse always sets some default for all options. This is attempting to strip those out so the default as determined by `Options.default` can be applied. I'm just trying to stick to a convention here where the default value specified for argparse for non-list parameters is `None` and for lists is `[]`, but I may be able to refactor the list to also be `None` which would simplify this condition.